app/team_test.go

```diff
@@ -489,9 +442,7 @@ func TestSanitizeTeam(t *testing.T) {
 		}
 
 		sanitized := th.App.SanitizeTeam(session, copyTeam())
-		if sanitized.Email != "" {
-			t.Fatal("should've sanitized team")
-		}
+		require.Equal(t, "", sanitized.Email, "should've sanitized team")
```

Comment:
nit: we can use `require.Empty` here to check for the empty string

Reply:
I preferred to use `Equal` because `Empty` won't fail if it's `nil`, for example.
Since `Email` is a string, we probably can rely on its type to not be nil.
That said, do you prefer `Equal` or `Empty`?